### DISCUSSION TITLE
"Cycling cadence sensor (Peloton compatability)" also controls heart rate?

### DISCUSSION THREAD

**Discussion by lychee128** [cagnulein/qdomyos-zwift#309] [Q&A] [answered]
I'm assuming this option determine whether the app transmits the cadence reading to the Peloton digital app.  However, I noticed that when I turn off this option, the heart rate data is also no longer transmitted to the Peloton app.  Is this correct?

Is there a way to keep the cadence data from being transmitted but still broadcast the heart rate data from the app to Peloton?

Thanks.

**Comment by cagnulein** [comment, marked as answer]
Yes you can use the virtual heart setting in the experimental settings

**Reply by lychee128** [reply]
Thanks.  That works great.
